pkg/api/helpers.go

```diff
@@ -494,15 +494,18 @@ func TolerationToleratesTaint(toleration *Toleration, taint *Taint) bool {
 }
 
 // TaintToleratedByTolerations checks if taint is tolerated by any of the tolerations.
-func TaintToleratedByTolerations(taint *Taint, tolerations []Toleration) bool {
-	tolerated := false
+func TaintToleratedByTolerations(taint *Taint, tolerations []Toleration) (bool, bool) {
```

Comment:
@davidopp I see, so I don't need to check `tolerations[i].Effect == TaintEffectNoScheduleNoAdmit` here.